tests/copp/test_copp.py

```diff
@@ -43,7 +43,7 @@
 from tests.common.fixtures.ptfhost_utils import change_mac_addresses      # noqa F401
 
 pytestmark = [
-    pytest.mark.topology("t0", "t1", "t2", "m0", "mx"),
+    pytest.mark.topology("t0", "t1", "t2", "m0", "m0-2vlan", "mx"),
```

Comment:
Is this change needed? I think m0 is ok?

Reply:
I tried without this change. All testcases are skipped. So I think it should be necessary.

Reply:
I see, we can set `--topoloty` to m0 when run test, then no need this change.

Reply:
Updated the code.